src/app/clusters/application-launcher-server/application-launcher-server.cpp

```diff
@@ -39,11 +39,56 @@
  ******************************************************************************/
 
 #include <app/Command.h>
+#include <app/clusters/application-launcher-server/application-launcher-server.h>
+#include <app/common/gen/af-structs.h>
+#include <app/common/gen/cluster-id.h>
+#include <app/common/gen/command-id.h>
+#include <app/common/gen/enums.h>
 #include <app/util/af.h>
 
+ApplicationLauncherResponse applicationLauncherClusterLaunchApp(EmberAfApplicationLauncherApp application, std::string data);
```

Comment:
Callbacks zapt, a followup.

Reply:
Issue [link](https://github.com/project-chip/connectedhomeip/issues/8086) for follow up.